server/models/rpaTaskModel.test.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-undef */
+const mongoose = require('mongoose');
+const { expect } = require('chai');
+const dbHandler = require('../utils/TestingUtils/testDatabaseHandler.js');
```

Comment:
There is a small typo: the `TestDatabaseHandler.js` starts with an uppercase Letter. Then the path should be resolveable.
```suggestion
const dbHandler = require('../utils/TestingUtils/TestDatabaseHandler.js');
```